cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -241,57 +247,51 @@
 #}
 
 {% block javascript scoped %}
-<!--[if gt IE 8]><!-->
-{# Include site-wide JavaScript. #}
-<script src="{{ static('js/routes/common.js') }}"></script>
-
-{# Check and include template-level JavaScript. #}
-{% macro template_js() %}
-    {% include 'js/routes/' + request.path.split('/')[1] + '/single.js' ignore missing %}
-{% endmacro %}
-{% set js_source = template_js() | trim %}
-{% if js_source | length > 0 %}
-<script src="{{ static('js/routes/' + request.path.split('/')[1] + '/single.js') }}"></script>
-{% endif %}
-
-{# Check and include component-level JavaScript. #}
-{% macro component_ondemand_js(js) %}
-    {% include 'js/routes/on-demand/' + js ignore missing %}
-{% endmacro %}
 
-{% if page and page.media %}
-    {% compress js %}
-    <script>
-    {% for js in page.media %}
-       {% set component_js_source = component_ondemand_js(js) | trim %}
-       {% if component_js_source | length > 0 %}
-            {{ component_js_source }}
-       {% endif %}
-    {% endfor %}
-    </script>
-    {% endcompress %}
-{% endif %}
-
-{# TODO: Remove the below block when all pages
-         using this template are moved to Wagtail. #}
-{# Check and include page-level JavaScript. #}
-{% macro page_js() %}
-    {% include 'js/routes/' + request.path[1:] + 'index.js' ignore missing %}
-{% endmacro %}
-{% set js_source = page_js() | trim %}
-{% if js_source | length > 0 %}
-<script src="{{ static('js/routes/' + request.path[1:] + 'index.js') }}"></script>
-{% endif %}
-<!--<![endif]-->
+{#
+  Modernizr adds a no-js class the HTML element if ECMAScript 5 features aren't
+  available. Check for that CSS class here and act accordingly.
+#}
+<script>
+    function loadScript(t,e){var n=document.createElement('script');n.type='text/javascript',n.onload=function(){return e?e():null},n.src=t,document.getElementsByTagName('head')[0].appendChild(n)}
```

Comment:
This is the unminified script:
```

function loadScript( url, callback ) {
  const script = document.createElement( 'script' );
  script.type = 'text/javascript';

  script.onload = function() {
    if ( callback ) {
      return callback();
    }
    return null;
  };

  script.src = url;
  document.getElementsByTagName( 'head' )[0].appendChild( script );
}
```

Reply:
Actually removed the callback onload bit, as none of the scripts use a callback. https://github.com/cfpb/cfgov-refresh/pull/3677/commits/25cf9ad6f58181863147a04378a7982f542d53a2